crates/oxc_linter/src/rules.rs

```diff
@@ -44,14 +49,18 @@ impl RuleEnum {
             Self::NoArrayConstructor(_) => Self::NoArrayConstructor(
                 maybe_value.map(NoArrayConstructor::from_configuration).unwrap_or_default(),
             ),
+            Self::NoExtraSemi(_) => Self::NoExtraSemi(
+                maybe_value.map(NoExtraSemi::from_configuration).unwrap_or_default(),
+            ),
         }
     }
 
-    pub fn run<'a>(&self, node: &AstNode<'a>, ctx: &LintContext<'a>) {
+    pub fn run<'a>(&self, node: &AstNode<'a>, ctx: &LintContext<'a>) -> Option<Fix> {
         match self {
             Self::NoDebugger(rule) => rule.run(node, ctx),
             Self::NoEmpty(rule) => rule.run(node, ctx),
             Self::NoArrayConstructor(rule) => rule.run(node, ctx),
+            Self::NoExtraSemi(rule) => rule.run(node, ctx),
```

Comment:
I thought no-debugger has a code fix as well, shall we defer implementing more rules?

Reply:
Oh, eslint doesn't have it, Rome does.